base/refpointer.jl

```diff
@@ -136,12 +141,41 @@ unsafe_convert(::Type{Ptr{T}}, r::Ref{NTuple{N,T}}) where {N,T} =
 unsafe_convert(::Type{Ptr{T}}, r::Ptr{NTuple{N,T}}) where {N,T} =
     convert(Ptr{T}, r)
 
-###
+### Methods for a Ref object that is backed by a object field
+"""
+    RefField <: Ref{T}
+
+Create an object reflection adaptor that represents the properties of a specific object field.
```

Comment:
Took me a while to parse this. Maybe

> Obtain a reference to a specific object field, by creating an object reflection adaptor that wraps the parent object and the properties of a specific object field.

Reply:
Or, to focus more on the interface which people will actually use:

```julia
    RefField(object, field)

Obtain a reference to an individual field of an `object`, where the given `field` may be an index or a symbol (see also [`Base.fieldindex`](@ref)).

The returned `RefField <: Ref` is an object reflection adaptor that wraps the parent `object` and the properties of a specific object field.
```